examples/single_file/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -eoux pipefail
```

Comment:
what is this for?

Reply:
I have this set for most of my bash scripts:

- `-e`: Makes the script exit on the first error (instead of continuing on the next line)
- `-o pipefail`: Exits if one command in the pipe fails (instead of continuing)
- `-u`: Prints to stderr and exits immediately if a variable does not exist
- `-x`: Prints each command to stderr before running it (good for logging/ci)

Reply:
well :D

Reply:
Will do!